litmus-portal/frontend/src/components/WorkflowCluster/styles.ts

```diff
@@ -42,17 +42,16 @@ const useStyles = makeStyles((theme: Theme) => ({
     marginRight: theme.spacing(2),
   },
   button: {
-    marginLeft: theme.spacing(20),
-    marginRight: theme.spacing(2),
+    marginLeft: theme.spacing(18),
   },
   buttonDiv: {
     marginTop: theme.spacing(6),
     display: 'flex',
     flexDirection: 'row',
   },
   or: {
-    marginTop: theme.spacing(1),
+    marginTop: theme.spacing(1.5),
   },
 }));
 
-export default useStyles;
+export default useStyles;
```

Comment:
new line deleted

Reply:
are these don't by prettier?...generally new line is added at the end of the file